server/conf/application.conf

```diff
@@ -274,10 +274,24 @@ play.ws {
 # libraryDependencies += cache
 #
 play.cache {
-  # If you want to bind several caches, you can bind the individually
-  #bindCaches = ["db-cache", "user-cache", "session-cache"]
+  # Specific caches can be injected using the @NamedCache annotation.
+  bindCaches = ["api-keys"]
 }
 
+## Security rules
+# https://github.com/pac4j/play-pac4j/wiki/Apply-security#2-protect-urls-via-the-securityfilter
+# ~~~~~
+# Specify authentication clients for request URL paths that match the regex.
+# If no regex matches, uses the default clients specified in SecurityModule.
```

Comment:
Suggestion: maybe tweak the phrasing: "uses the default clients" as I find it a little unclear if the below 5 lines of config are doing this or if it's an implicit aspect of our overall setup.

There's no grammatical subject before "uses" so the reader has to mentally fill it in with Pac4j, CiviForm, the 5 lines below etc.

Reply:
updated